app/src/main/java/io/islnd/android/islnd/app/database/IslndDbHelper.java

```diff
@@ -85,6 +85,13 @@ public void onCreate(SQLiteDatabase db) {
 
                 " UNIQUE (" + IslndContract.ProfileEntry.COLUMN_USER_ID + ") ON CONFLICT REPLACE);";
 
+        final String SQL_CREATE_NOTIFICATION_TABLE = "CREATE TABLE " + IslndContract.NotificationEntry.TABLE_NAME + " (" +
+                IslndContract.NotificationEntry._ID + " INTEGER PRIMARY KEY," +
+                IslndContract.NotificationEntry.COLUMN_NOTIFICATION_USER_ID + " INTEGER NOT NULL," +
+                IslndContract.NotificationEntry.COLUMN_NOTIFICATION_TYPE + " INTEGER NOT NULL," +
+                IslndContract.NotificationEntry.COLUMN_POST_ID + " TEXT NOT NULL, " +
```

Comment:
Makes sense. The postid might be a little weird since it's a composite key with the userid in the post table. We could store the post userid as well, but we know that every post will be the clients. Not sure what to do there.